### PR TITLE
Changed strings representing LaTeX formulas to raw strings in LaTeX tests.

### PR DESCRIPTION
**Pull request by costika1234**
#### Brief description of what is fixed or changed
Improved consistency and readability of LaTeX tests by converting strings representing LaTeX formulas (e.g. `"\\frac{1}{2}"`) to their raw counterparts (`r"\frac{1}{2}"`). 

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by costika1234** [comment]
@asmeurer Would you be able to have a look on this PR?

File: sympy/printing/tests/test_latex.py [hunk 1]
```diff
@@ -1984,7 +1975,7 @@ def test_translate():
 def test_other_symbols():
     from sympy.printing.latex import other_symbols
     for s in other_symbols:
-        assert latex(symbols(s)) == "\\"+s
+        assert latex(symbols(s)) == r"" "\\" + s
```
Comment (oscarbenjamin) [inline sympy/printing/tests/test_latex.py:1978]: Is this a typo?

**Comment by oscarbenjamin** [comment]
I reviewed this and it looks good. Thanks!

I left one question though...

File: sympy/printing/tests/test_latex.py (hunk 1 above)
Comment (costika1234) [reply sympy/printing/tests/test_latex.py:1978]: It is not. I just wanted to be consistent and use a raw string in that scenario too, however, `r"\"` is invalid in Python. See https://docs.python.org/3/faq/design.html#why-can-t-raw-strings-r-strings-end-with-a-backslash.

Comment (oscarbenjamin) [reply sympy/printing/tests/test_latex.py:1978]: Oh yes, I forgot about that. Still I don't think that the empty rawstring is needed though if it isn't actually doing anything. If we have to use `"\\"` then that's fine.

Comment (costika1234) [reply sympy/printing/tests/test_latex.py:1978]: That's right, we don't need that `r""` bit, but for the sake of consistency I would include it. I also have it here: https://github.com/sympy/sympy/pull/20915/files#diff-d909b0940fe23d80682ae4319261f194ca445b99359428612db7f13ae8da7d2cR299. Let me know if you want this to be modified (either here or in both places) and I will update the PR.

**Comment by oscarbenjamin** [comment]
No, I think this is good. Thanks!

**Pull request merged**